### DISCUSSION TITLE
Join the supabase-kt Kotlin Slack channel

### DISCUSSION THREAD

**Discussion by jan-tennert** [supabase-community/supabase-kt#524] [Announcements]
There is now a [supabase-kt channel](https://kotlinlang.slack.com/archives/C06QXPC7064) in the kotlinlang Slack.

**Comment by aarchish** [comment]
Is this a public discussion Slack Channel?
or is it for internal development?

**Reply by jan-tennert** [reply]
Its public.

**Reply by aarchish** [reply]
I get an error saying that my email id is not assigned to the workspace

**Reply by jan-tennert** [reply]
Checkout https://kotlinlang.org/community/ for the Kotlin Slack. `supabase-kt` is just a channel in the JetBrains hosted Kotlin Slack.
